analyzer/pgn.h

```diff
@@ -3141,18 +3141,20 @@ Pgn pgnList[] = {
       {"SID", BYTES(1), 1, false, 0, ""},
       {0}}}
 
+    /* https://www.nmea.org/Assets/20140102%20nmea-2000-127509%20pgn%20corrigendum.pdf */
     ,
     {"Inverter Status",
      127509,
      PACKET_COMPLETE,
      PACKET_SINGLE,
      4,
      0,
-     {{"Instance", BYTES(1), 1, false, 0, ""},
+     {{"Inverter Instance", BYTES(1), 1, false, 0, ""},
```

Comment:
Why 'Inverter Instance'. All other Messages translate the NMEA2000 instance into 'Instance'. 
I can't see why 127509 should be an exception here.  Would be nice to be changed back to
'Instance' (not only because it breaks my code ;-)   )

Reply:
It's probably because I didn't have live data for this PGN, so I tried to match the [source](https://www.nmea.org/Assets/20140102%20nmea-2000-127509%20pgn%20corrigendum.pdf) document as closely as possible. Maybe I thought an "Instance" field would be ambiguous since the PGN also contains "AC Instance" and "DC Instance". That being said, I agree that it should stay consistent with other PGNs, so if you submit a pull request for this you'll have my support.

Reply:
I agree with @patg.

Reply:
Fixed in master.